project/plugins/__init__.py

```diff
@@ -0,0 +1,59 @@
+import json
```

Comment:
This whole file is incorrect.  This should be an empty file.  Somehow you've added plugin code into this file by mistake